wgpu-types/src/lib.rs

```diff
@@ -3122,6 +3122,19 @@ pub struct SurfaceConfiguration {
     pub alpha_mode: CompositeAlphaMode,
 }
 
+impl Default for SurfaceConfiguration {
+    fn default() -> Self {
```

Comment:
I think I'd be more happy with a builtin `from_size` function that takes width and height and fills in the rest.